extension.js

```diff
@@ -42,7 +42,7 @@ class ExtensionDBusInterface {
 
         Main.wm.skipNextEffect(current_window.get_compositor_private());
         current_window.unmaximize(Meta.MaximizeFlags.VERTICAL);
-
+        current_window.unmaximize(Meta.MaximizeFlags.HORIZONTAL);
```

Comment:
This method is called by the application before mouse resizing begins (I guess I should have called it `PrepareForResize` or something like that).
For vertical resizing, `unmaximize()` should be called with `Meta.MaximizeFlags.VERTICAL`. For horizontal resizing - `Meta.MaximizeFlags.HORIZONTAL`. Unmaximizing in both directions at once is not necessary (and maybe it even causes some issues, but I haven't found any so far).
Although there is no horizontal resizing with mouse yet, so for now this line can be simply removed, and then added later, together with horizontal mouse resizing.

Reply:
Removed.